dom0/sd-sys-vms.sls

```diff
@@ -12,11 +12,10 @@ include:
 {% set sd_supported_fedora_version = 'fedora-35' %}
 
 
-# Install latest templates required for SDW VMs.
-dom0-install-fedora-template:
-  cmd.run:
-    - name: >
-        qvm-template install {{ sd_supported_fedora_version }}
+# Ensure the latest templates required for SDW VMs are installed.
+dom0-fedora-template-is-installed:
```

Comment:
Multiple subsequent commands `require` this one under the old name (`dom0-install-fedora-template`), so this Salt state will fail with "requisites were not found" error unless all those `require` conditions are updated, e.g.:

https://github.com/freedomofpress/securedrop-workstation/blob/b7eebfe3f54db6422afc3ba7e7d502bbc2a24cc6/dom0/sd-sys-vms.sls#L28-L29

Reply:
(Also note that all those `cmd` instances will need to be changed to `qvm`.)

Reply:
Thanks for the review and suggestions @eloquence :+1: 

I may wait until I'm in a position to test this before looking into it further, especially if #797 is unconfirmed/unclear at this point.

Reply:
Sounds good. In the meantime I've confirmed that with those fixes, it runs without errors and is idempotent. So I think this would be a neutral change.